.gitmodules

```diff
@@ -1,12 +1,12 @@
 [submodule "packages/simulator-server"]
 	path = packages/simulator-server
-	url = git@github.com:software-mansion-labs/simulator-server.git
+	url = https://github.com/software-mansion-labs/simulator-server.git
```

Comment:
AFAIK https links requires different type of setup locally when you want the repositories to be writeable. This means that while it may be read on CI it will cause problems on dev installations where people wouldn't be able to make updates directly from submodules (unless they set up HTTPS properly on their computers which I don't know how can be done).

I'd prefer if we could keep this file unchanged assuming there's some easy easy way to make it work. I quickly found that you can pass `submodules` option when loading main repo: https://github.com/marketplace/actions/checkout-submodules#alternatives

Reply:
The problem is it doesn't work with SSH without any additional setup, passing `submodules` has to be ruled out as `simulator-server` is private and we don't want to pull it, as well as the other, non-essential for CI submodules that take too long to pull.

Reply:
can we list the submodules we want to checkout in the action then? There's another example under the link I sent that does this:
```
steps:
- name: Checkout repository
  uses: actions/checkout@v2
- name: Checkout submodules
  run: git submodule update --init --recursive
```

So here instead of running wildcard update we can list submodules we want to pull

Reply:
^ This works only with HTTPS modules, if we want to keep SSH we need [some shenanigans](https://stackoverflow.com/a/60227822).
I think it's better to keep the submodules we need to check out in one place (npm scripts). In https://github.com/software-mansion/radon-ide/commit/33b4e2c89e9c82c2426e1257228ed9b2622a417f, I've added a workaround with setting git config to use HTTPS on CI and that resolves the issue with SSH submodules.